microhapdb/tests/test_cli.py

```diff
@@ -138,40 +138,48 @@ def test_main_marker_query(capsys):
-    mh19SHY-002        9     165 chr19 55588421 55588585 3.544                   Wu2021
-     mh19WL-027        4     193 chr19 55796289 55796481 4.544                   Yu2022
-     mh19WL-012       11     195 chr19 57013159 57013353 4.385                   Yu2022
-
+           Name  NumVars  Extent Chrom    Start      End    Ae                              Source
+  mh19WL-007.v3        6     251 chr19   410610   410860 1.126                            Yu2022G3
+  mh19WL-007.v1        5     102 chr19   410759   410860 1.000                   Yu2022G1;Yu2022G2
+  mh19WL-007.v2        4      89 chr19   410772   410860 1.000                            Yu2022G4
+   mh19USC-19pA        3      38 chr19   561779   561816 2.924                      delaPuente2020
+     mh19WL-024        6     198 chr19  1955525  1955722 4.977                            Yu2022G1
+     mh19WL-025        4     170 chr19  3120344  3120513 4.746                            Yu2022G1
+     mh19KK-056        2     201 chr19  4852125  4852325 2.608                            Kidd2018
+ mh19SHY-001.v1        8     185 chr19  7698913  7699097 6.469                              Wu2021
+ mh19SHY-001.v2        5     106 chr19  7698992  7699097 6.442                   Yu2022G1;Yu2022G2
+ mh19SHY-001.v4        6     280 chr19  7698992  7699271 9.195                            Yu2022G3
+ mh19SHY-001.v3        4      67 chr19  7699031  7699097 4.488                            Yu2022G4
+     mh19WL-026        4      84 chr19  9009766  9009849 3.955                            Yu2022G1
+     mh19CP-007        3      42 chr19 14310740 14310781 3.271                            Kidd2018
+     mh19WL-011        3      96 chr19 15931448 15931543 1.803 Yu2022G1;Yu2022G2;Yu2022G3;Yu2022G4
+   mh19USC-19pB        5      66 chr19 16040865 16040930 3.866                      delaPuente2020
+    mh19ZHA-006        6      63 chr19 20579863 20579925 3.175                             Sun2020
+      mh19NH-23        3      95 chr19 22052724 22052818 2.032                         Hiroaki2015
+  mh19KK-299.v1        5     154 chr19 22546698 22546851 4.236                 Kidd2018;Turchi2019
+  mh19KK-299.v2        7     154 chr19 22546698 22546851 4.253                        Gandotra2020
+  mh19KK-299.v4       10     182 chr19 22546698 22546879 4.253                         Pakstis2021
+  mh19KK-299.v3        3      63 chr19 22546749 22546811 3.771                         Staadig2021
+    mh19ZHA-007        4     141 chr19 28397316 28397456 4.435                 Kureshi2020;Sun2020
+mh19USC-19qA.v1        4      46 chr19 33273772 33273817 3.660                      delaPuente2020
+mh19USC-19qA.v3        5      49 chr19 33273772 33273820 5.084                            Yu2022G4
+mh19USC-19qA.v2        6     185 chr19 33273772 33273956 7.147          Yu2022G1;Yu2022G2;Yu2022G3
+     mh19WL-022        4     283 chr19 39678442 39678724 3.975                            Yu2022G3
```

Comment:
This is a great microcosm of all the complicated Venn diagrams of microhaps being defined and redefined within and between studies.